tenseal/tensors/ckkstensor.py

```diff
@@ -87,3 +88,7 @@ def sum_batch(self) -> "CKKSTensor":
     def sum_batch_(self) -> "CKKSTensor":
         self.data.sum_batch_()
         return self
+
+    def reshape(self, shape: List[int]):
+        "Changes the internal representation to a new shape"
+        return self.data.reshape(shape)
```

Comment:
I think reshape in most tensor library outputs a new tensor objects. Should we also have reshape (copy) and reshape_ (inplace) for this?

Reply:
I'm not talking about the wrapper but the Cpp function